python/lsst/daf/butler/core/timespan.py

```diff
@@ -376,10 +633,12 @@ class _CompoundTimespanDatabaseRepresentation(TimespanDatabaseRepresentation):
 
     Parameters
     ----------
-    begin : `sqlalchemy.sql.ColumnElement`
-        SQLAlchemy object representing the begin (inclusive) endpoint.
-    end : `sqlalchemy.sql.ColumnElement`
-        SQLAlchemy object representing the end (exclusive) endpoint.
+    begin_nsec : `sqlalchemy.sql.ColumnElement`
+        SQLAlchemy object representing the lower (inclusive) bound, as a
+        64-bit integer column containing nanoseconds since ``time_utils.EPOCH`.
+    end_nsec : `sqlalchemy.sql.ColumnElement`
+        SQLAlchemy object representing the upper bound (exclusive), as a 64-bit
+        integer column containing nanoseconds since `time_utils.EPOCH`.
```

Comment:
Constructor has a single `nsec` parameter which is a tuple, and also `name` which is not documented here.